### PR TITLE
Add create_struct_mock macro to mock structs

### PR DESCRIPTION
**Pull request by werner**
Hi, I wish I can use this to mock Time.now.

Best regards.

### CODE REVIEW

**Comment by waterlink** [comment]
Hi @werner 

Thanks for the PR!

LGTM

**Comment by waterlink** [comment]
I have released version `0.5.0` with your changes.

File: spec/mocks_spec.cr [hunk 1]
```diff
@@ -125,13 +135,20 @@ describe Mocks do
       Example.hello_world("halo").should eq("halo there world")
     end
 
-    it "works with module methdods" do
+    it "works with module methods" do
```
Comment (waterlink) [inline spec/mocks_spec.cr:138]: :+1: for noticing a typo!

Comment (marceloboeira) [inline spec/mocks_spec.cr:138]: I guess it was my fault, sorry :disappointed:

Comment (waterlink) [inline spec/mocks_spec.cr:138]: @marceloboeira No worries!

**Pull request merged**